File: serverH/server/ServiceOperationImpl.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace serverH
{
   class ServiceOperationImpl: MarshalByRefObject, ServiceOperation
    {
        public static List<Service> lstService = new List<Service>();

        public List<Service> getListOfService()
        {
            return lstService;
        }

        public List<Service> addNewService(Service item)
        {
            lstService.Add(item);
            return lstService;
        }
        public List<Service> delService(int index)
        {
            lstService.RemoveAt(index);
            return lstService;
        }
        public int getSumOfService()
        {
            int sum = 0;
            foreach (Service service in lstService)
                sum += service.getPrice();
            return sum;
        }
    }
}

File: Server/src/operation/ServiceOperationImpl.java
package operation;

import types.Service;

import java.util.ArrayList;
import java.util.List;

public class ServiceOperationImpl implements ServiceOperation{

    static List<Service> lstService = new ArrayList<Service>();

    static{
        lstService.add(new Service("Услуга 1","Хостел 2",11,500,"10 минут","Червеченко А.А."));
        lstService.add(new Service("Услуга 2","Хостел 1",23,700,"30 минут","Любовцева М.Г."));
        lstService.add(new Service("Услуга 3","Хостел 3",41,250,"5 минут","Михалкова Н.Я."));
    }

    @Override
    public List<Service> getListOfService() {
        return lstService;
    }

    @Override
    public List<Service> addNewService(Service service){
        lstService.add(service);
        return lstService;
    }
    @Override
    public List<Service> delService(int index)
    {
        lstService.remove(index);
        return lstService;
    }
    @Override
    public int getSumOfService(){
        int sum = 0;
        for(Service service: lstService)
            sum += service.getPrice();
        return sum;
    }
}
